### PR TITLE
#167891569 Setup API Documentation Using Swagger

### PR DESCRIPTION
**Pull request by thatkazuk1**
#### What does this PR do?
- setup Barefoot Nomad API documentation.
- added .gitattributes file to handle EOF conflict with git.
- added swagger.json API documentation (more to be done later).

#### Description of Task to be completed?
Nil

#### How should this manually be Tested?
* git clone `https://github.com/andela/banshee-bn-backend`
* `cd` into `banshee-bn-backend`
* Run `npm i`
* Run `npm devstart`
* Open browser and navigate to `localhost:3000/docs`

#### What are the relevant Pivotal Tracker stories?
#167891569

#### Screenshots (if appropriate)
barefoot0

### CODE REVIEW

File: swagger.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        "summary": "The root endpoint",
+        "responses": {
+          "200": {
+            "description": "Success"
+          }
+        }
+      }
+    }
+  }
+}
```
Comment (tvpeter) [inline swagger.json]: Seems an extra line is needed at the end of the file here.

**Review by tvpeter — APPROVED** [review]
Awesome job. Can you please add `/docs` route to the readme file as the api documentation. Thank you.

File: swagger.json (hunk 1 above)
Comment (thatkazuk1) [reply swagger.json]: Yeah. Will insert that immediately

**Review by max-wel — COMMENTED** [review]
Looks good to me

**Review by femitj — COMMENTED** [review]
nice @meetKazuki

**Review by femitj — APPROVED** [review]

**Review by max-wel — APPROVED** [review]

**Comment by thatkazuk1** [comment]
@daniellamarr conflict has been resolved

**Review by daniellamarr — APPROVED** [review]

**Pull request merged**